tests/expected_cmdline_output_files/basic_connectivity_expl_output.txt

```diff
@@ -0,0 +1,14775 @@
+final fw rules for query: , config: testcase8-networkpolicy1.yaml:
+src: 0.0.0.0/0 dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+<entry dst="0.0.0.0/0" id="1" src="0.0.0.0/0">
+ <text>
+  Configurations affecting 0.0.0.0/0:
+IP blocks have no configurations
+ </text>
+</entry>
+<entry dst="default/cog-agents(DaemonSet)" id="2" src="0.0.0.0/0">
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)default/cog-agents(DaemonSet):
+
+(src)0.0.0.0/0:
+IP blocks have no configurations
+
+(dst)default/cog-agents(DaemonSet):
+Policy Configurations:
+default/cog-agents(DaemonSet): line 5 in file ./k8s_testcases/example_podlist/pods_list.json
```

Comment:
this is not the policy configuration, it's the node configuration

Reply:
Fixed.